src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/OpphørValideringService.kt

```diff
@@ -38,13 +39,25 @@ class OpphørValideringService(
         revurderFra: LocalDate?,
     ) {
         brukerfeilHvis(revurderFra == null) { "Revurder fra dato er påkrevd for opphør" }
+
         beregningsresultatTilsynBarn.perioder.forEach { periode ->
             periode.beløpsperioder.forEach {
                 brukerfeilHvis(it.dato >= revurderFra) { "Opphør er et ugyldig vedtaksresultat fordi det er utbetalinger på eller etter revurder fra dato" }
             }
         }
     }
 
+    fun validerIngenUtbetalingEtterRevurderFraDato(
```

Comment:
Denne er ikke brukt

Reply:
Siden det ikke gjennomføres beregning for Opphør så har man heller intet beregningsresultat å validere. Sletter "validerIngenUtbetalingEtterRevurderFraDato"

Reply:
b55eca87bcceab7fbf747ebd87825dcdf67129e9